GVRf/Framework/jni/shaders/post_effect_shader_manager.h

```diff
@@ -53,20 +52,22 @@ class PostEffectShaderManager: public HybridObject {
     }
 
     ~PostEffectShaderManager() {
+        delete color_blend_post_effect_shader_;
+        delete horizontal_flip_post_effect_shader_;
+        // We don't delete the custom shaders, as their Java owner-objects will do that for us.
```

Comment:
Imho, I think in the destructor you have to do "delete" with all the pointers in std containers such as custom_post_effect_shaders_, otherwise only those pointers will be released but the content those pointers point to will become memory leak.